src/browser/modules/EditorInfo/DisplayRelationshipDirection.jsx

```diff
@@ -68,8 +75,16 @@ function DisplayRelationshipDirection (props) {
             }}
             value={selectedDirection}
             onChange={e => {
-              setSelectedDirection(e.target.value)
-              setButtonVisibility(true)
+              if (
+                e &&
+                e.target.value !== propsToStateMap(props.relationshipEndpoint)
+              ) {
+                setSelectedDirection(e.target.value)
+                setButtonVisibility(true)
+              } else {
+                setSelectedDirection(e.target.value)
+                setButtonVisibility(false)
```

Comment:
what if direction is not changed but other field values are changed. e.g. relationship name.
this setButtonVisibility(false) will switch off confirmation button unnecessarily.